code/style.css

```diff
@@ -1,4 +1,185 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/* BODY */
+
+body {
+  background: rgb(231, 103, 89);
+  color: rgb(76, 223, 76);
+  font-family: 'Potta One';
+  margin: 0;
+}
+
+/* HEADER */
+
+header {
+  background: white;
+  width: 100%;
+  height: 100px;
+  display: flex; 
+  align-items: center;
+  padding: 0 16px;
+}
+
+header img {
+  display: flex;
+  width: 50px;
+  height: 50px;
+  margin-right: 24px;
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  color:  rgb(76, 223, 76);
+  text-decoration: none;
+  margin-left: 50px;
+}
+
+/* NEWS */
+
+.news-page {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.news-page img {
+  width: 100%;
+  height: auto;
+}
+
+.news-card1 {
+  width: 100%;
+  height: 100px;
+  background: white;
+  color: black;
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+.news-card2 {
+  width: 90%;
+  height: 100px;
+  background: white;
+  color: black;
+  margin: 10px;
+  border-radius: 24px;
+}
+
+.news-card3 {
+  width: 90%;
+  height: 100px;
+  background: white;
+  color: black;
+  margin: 10px;
+  border-radius: 24px;
+}
+
+.news-card4 {
+  width: 90%;
+  height: 100px;
+  background: white;
+  color: black;
+  margin: 10px;
+  border-radius: 24px;
+}
+
+.news-card5 {
+  width: 90%;
+  height: 100px;
+  background: white;
+  color: black;
+  margin: 10px;
+  border-radius: 24px;
+}
+
+.news-card6 {
+  width: 90%;
+  height: 100px;
+  background: white;
+  color: black;
+  margin: 10px;
+  border-radius: 24px;
+}
+
+.news-card7 {
+  width: 90%;
+  height: 100px;
+  background: white;
+  color: black;
+  margin: 10px;
+  border-radius: 24px;
+}
```

Comment:
Since these all have the same styling I would suggest that you instead gave all these elements the same class name to not have to write the same code over and over again :)